lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
If it's working properly _now_, I'm not really sure that's worth the effort, especially if this is the method that other packages use to deal with this issue.